client/payment-methods/index.js

```diff
@@ -62,6 +76,55 @@ const methodsConfiguration = {
 	},
 };
 
+// @todo - remove once #2174 is merged and use real banner instead.
+function UPESetupBanner( { setIsUpeEnabled } ) {
```

Comment:
Rather than prop-drill the flag, the context/hook can be leveraged directly
```suggestion
function UpeSetupBanner() {
	const [ , setIsUpeEnabled ] = useIsUpeEnabled();

```


(minor) Since the JS components are already using `Upe` (instead of `UPE`), for sake of consistency it'll be better to rename the new components introduced in this file to use `Upe` in their naming